test/backingstore/rev-backingstore.py

```diff
@@ -15,86 +15,86 @@
 DEBUG_MEM = 10
 DEBUG_LEVEL = 10
 VERBOSE = 10
-MEM_SIZE = 1024*1024*1024-1
+memSize = 1024*1024*1024-1
 
 # Define the simulation components
 comp_cpu = sst.Component("cpu", "revcpu.RevCPU")
 comp_cpu.addParams({
-        "verbose" : 5,                                # Verbosity
-        "numCores" : 1,                               # Number of cores
-        "clock" : "2.0GHz",                           # Clock
-        "memSize" : MEM_SIZE,                         # Memory size in bytes
-        "machine" : "[0:RV64GC]",                      # Core:Config; RV64G for core 0
-        "startAddr" : "[0:0x00000000]",               # Starting address for core 0
-        "memCost" : "[0:1:10]",                       # Memory loads required 1-10 cycles
-        "program" : os.getenv("REV_EXE", "backingstore.exe"),  # Target executable
-        "enable_memH" : 1,                            # Enable memHierarchy support
-        "splash" : 1                                  # Display the splash message
+        "verbose": 5,                                         # Verbosity
+        "numCores": 1,                                        # Number of cores
+        "clock": "2.0GHz",                                    # Clock
+        "memSize": memSize,                                   # Memory size in bytes
+        "machine": "[0:RV64GC]",                              # Core:Config; RV64G for core 0
+        "startAddr": "[0:0x00000000]",                        # Starting address for core 0
+        "memCost": "[0:1:10]",                                # Memory loads required 1-10 cycles
+        "program": os.getenv("REV_EXE", "backingstore.exe"),  # Target executable
+        "enableMemH": 1,                                      # Enable memHierarchy support
+        "splash": 1                                           # Display the splash message
```

Comment:
I have added spaces so that the `#` comments on the right all line up on the same column, which must be two spaces away from the longest line.